src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -9,6 +9,1404 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public partial class IOperationTests : SemanticModelTestBase
     {
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void TargetTypedObjectCreationWithMemberInitializers()
```

Comment:
Thanks for adding those IOperation tests!
Tagging @333fred @mavasani in case they want to take a look at those tests too.